week-1/Homework/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,14 @@
-// Write your function here
 
-var greeting = createGreeting("Daniel");
+
+function createGreeting (name) {
+    const message = "Hello, my name is " + name;
+    return message;
+}
```

Comment:
Have a look at the functions in the previous exercises, you didn't need a variable to return the function output. You can do the same here and save some memory:
```
function createGreeting (name) {
    return "Hello, my name is " + name;
}
```